extensions/client-batch.md

```diff
@@ -28,14 +28,23 @@ batch type, but is designed as a framework for other specifications.
 Certain specifications call for clients to be able to send batched messages to
 servers, for instance [`draft/multiline`][]
 
-## Capabilities
+## Architecture
+
+### Dependencies
+
+This specification does not depends on the [`batch`][] capability,
+as it is only used for server-initiated batches.
```

Comment:
does not depend *[no s]* on the batch capability, which is used for server-initiated batches only.